e2e-tests/src/PM/helper.js

```diff
@@ -0,0 +1,39 @@
+
+export const onboardCardMethodWithError4xx = async (cardData) => {
+    await fillCardForm(cardData);
+    await page.waitForNavigation();
+    const url = await page.url();
+    return url;
+};
+
+export const onboardCardMethodWithError5xx = async (cardData) => {
```

Comment:
Maybe a more appropriate name could be something like `onboardCardMethodWithNetworkError`. This because when offline a fetch promise rejects and we have no response

Reply:
Great idea 🙂

Reply:
Fixed with 7870d30f9602bc82ab478806b3e6273c1cbfc4c8